GetPhrase.js

```diff
@@ -0,0 +1,26 @@
+var CreatePhrase = require("./CreatePhrase");
+var fs = require("fs");
+
+var GetPhrase = function(Word) {
+  this.arr = [];
+  fs.readFile("phrases.txt", "utf8", function(err, phrases) {
```

Comment:
In this instance using readFileSync would be more effective against bugs and most of the logic would remain the same.